src/views/ManageList.jsx

```diff
@@ -1,24 +1,28 @@
 import { useCallback, useState } from 'react';
 import { useStateWithStorage } from '../utils';
-import { addItem } from '../api';
+import { addItem, shareList } from '../api';
 
 const soonDate = 7;
 const kindOfSoonDate = 14;
 const notSoonDate = 30;
 
-export function ManageList() {
+export function ManageList({ userId }) {
 	const [daysUntilNextPurchase, setDaysUntilNextPurchase] = useState(null);
 	const [itemName, setItemName] = useState('');
+	const [emailData, setEmailData] = useState('');
 
 	const [listPath, setListPath] = useStateWithStorage(
```

Comment:
Since `setListPath` isn't being used here, we could remove it!

Reply:
fixed, thank you!